src/main/resources/META-INF/resources/primefaces/ai/smartcam/smartcam.js

```diff
@@ -0,0 +1,122 @@
+PrimeFaces.widget.SmartCam = PrimeFaces.widget.BaseWidget.extend({
+	
+	init: function(cfg) {
+    	this._super(cfg);
+        if(this.cfg.disabled) {
+            return;
+        }
+        
+        this.cfg.width = this.cfg.width || 320;
+        this.cfg.height = this.cfg.height || 240;
+        this.cfg.renderTimeout = this.cfg.renderTimeout || 50;
+        
+        if (!("autoStart" in this.cfg)) {
```

Comment:
You don't need to initialize this autostart variable at all just say later in the code `if (this.cfg.autoStart)` and its false by default and will only ever be true if you passed in true.  Javascript will assume false even if the variable is undefined.